runelite-client/src/main/java/net/runelite/client/plugins/keyremapping/KeyRemappingConfig.java

```diff
@@ -241,4 +241,15 @@ default ModifierlessKeybind esc()
 	{
 		return new ModifierlessKeybind(KeyEvent.VK_ESCAPE, 0);
 	}
+
+	@ConfigItem(
+			position = 20,
+			keyName = "space",
+			name = "Space",
+			description = "The key which will replace {Space}."
```

Comment:
Your indents here are one level too far.
```suggestion
		position = 20,
		keyName = "space",
		name = "Space",
		description = "The key which will replace {Space}."
```